### ISSUE TITLE
We are missing title & description meta tags

### ISSUE THREAD

**Issue by jasonify** [SoftwareEngineeringDaily/sedaily-front-end#474]
For episodes we want to add title + description meta tags:

`<meta data-vmid="og:title" property="og:title" content="Stripe Observability Pipeline with Cory Watson" data-vue-meta="true">`

**Comment by jasonify** [comment]
hey @TheHollidayInn  do you mind providing more detail, @joshmorel  is looking into this but he is saying he is seeing this in prod?

**Comment by joshmorel** [comment]
Yeah here's what I see, for example, with "Stripe Observability..." for example. There may be some intricacy for Google SEO crawling I'm not aware of however...

![image](https://user-images.githubusercontent.com/10546720/39155989-5c918bba-4722-11e8-8f26-c6a1c70267bc.png)

**Comment by jasonify** [comment]
@joshmorel  Here is what I'm seeing:

`
<!DOCTYPE html><html><head><meta charset=utf-8><title>Software Daily</title><meta name=viewport content="width=device-width,initial-scale=1"><meta name=fragment content=!><link rel="shortcut icon" type=image/png href=/static/favicon.png><link href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700" rel=stylesheet><style>.tribute-container li {
      background: white;
      padding: 10px 20px;
      list-style: none;
      border-bottom: 1px solid grey;
    }
    ul.tribute-container {
    }
    .tribute-container li.highlight {
      background: cyan;
      padding: 10 20px;
    }

    .tribute-container li:hover {
      background: grey;
      padding: 10 20px;
    }

    .tribute-container li.no-match {
      display: none;
    }</style></head><body><div id=app></div><script src=https://js.stripe.com/v3/ ></script><script type=text/javascript src=/static/js/vendor.ad06170a20cc84de269b.js></script><script type=text/javascript src=/static/js/main.297c1b81072885295f4f.js></script></body></html>
`

When I:
`
curl https://www.softwaredaily.com/post/5adf0e5e39351b00040d1ec2/NATS-Messaging-with-Derek-Collison >> test.html`

**Comment by TheHollidayInn** [comment]
Curling won't work because you we aren't doing serverside rendering. Google parses javascript.

The title tag is there, it just isn't the name of the podcast. You should use Vue meta like this:https://github.com/SoftwareEngineeringDaily/sedaily-front-end/blob/ce9644abd13e3a3ea843fbf6a5cd1355da10527c/src/views/JobView.vue#L125 for podcasts, but you also need to override the title and description.

**Comment by joshmorel** [comment]
Okay, now I'm following! The original issue body text threw me off a bit, I was thinking the reference was to `og:title`, `og:description` meta tags. 

But to confirm what we need are an overridden `<title>` tag and a meta tag with `itemprop="description"` attribute.

This is an easy addition. Anything else that would be helpful for Google search while I'm at it?

**Comment by TheHollidayInn** [comment]
Yep! Except you also need the description tag. Not sure you need the itemprop attribute. https://moz.com/learn/seo/meta-description

**Comment by joshmorel** [comment]
I saw itemprop in the WordPress site, it seems to be related to [product specific search](https://developers.google.com/search/docs/data-types/product) so maybe not so relevant for us (or maybe it is and I should just include it?).

Since they're different attributes but all similar in purpose should I just use one meta tag for social sharing and Google SEO description tag ? E.g.:

```html
<meta 
  data-vmid="description" 
  property="og:description" 
  name="description" 
  itemprop="description" 
  content="Stripe processes payments for thousands of businesses. A single payment could involve 10 different networked services. If a payment fails, engineers need to be able to diagnose what happened. The root cause could lie in any of those services. Distributed tracing is used to find the causes of failures and latency within networked services. In a distributed trace, each period of time associated w..."
  data-vue-meta="true"
>
```

Or should I make them separate with same content?

**Comment by TheHollidayInn** [comment]
I'm not sure if the attributes should work, but I'd just create separate tags. So for example on the page I linked:
```
meta: [
        this.metaTag('og:title', this.job.title),
        this.metaTag('og:url', location.href),
        this.metaTag('og:description', this.jobDescriptionSummary)
      ]
```

We just need 
```
 this.metaTag('title', this.job.title),
 this.metaTag('description', this.job.title),
```

Added. So, all of that should be on the post item. I'm not seeing it, but I also haven't looked at the code in a while.

**Comment by joshmorel** [comment]
I'll put in a PR but I needed to branch off of [this first](https://github.com/EmilyRosina/sedaily-front-end/pull/1) otherwise `develop` branch currently does not work on Linux at all.

Should I wait for #471 to be resolved or just make a PR off of the branch's branch?

**Comment by TheHollidayInn** [comment]
@jasonify thoughts?

**Comment by jasonify** [comment]
I was thinking we should resolve the post->Post in a small quick tiny PR anyway so that sounds good to me.  I don't see why it needs to  be coupled with the other changes in that PR.
